File: areas.m
  fprintf("calculadora de areas para:\n")
  fprintf("1.Cuadrado.\n")
  fprintf("2.Circulo.\n")
  fprintf("3.Triángulo.\n")
  fprintf("4.Rectángulo.\n")
  fprintf("otro para Salir\n")
  fprintf("****************************\n")
  x=input("Escoja la figura: ")

  fid = fopen('areas.txt','a');
  if x==1
      L=input('Ingrese el lado: ')
      area=L**2
      fprintf('el area del cuadrado es: %d', area)
      fprintf(fid,"\nCuadrado de lado %d tiene de area %d",L,area)
    
  elseif x==2
      R=input('Ingrese el radio del circulo: ')
      area=pi*R*R
      fprintf('el area del circulo es %d', area)
      fprintf(fid,"\nCirculo de radio %d tiene de area %d",R,area)
    
  elseif x==3  
      b=input('Ingrese la base: ')
      h=input('Ingrese la altura: ')
      area=b*h/2
      fprintf('El area del triángulo es: %d', area)
      fprintf(fid,"\nTriangulo de base %d y altura %d tiene de area %d",b,h,area)
    
  elseif x==4
      b=input('Ingrese la base: ')
      h=input('Ingrese la altura: ')
      area=b*h
      fprintf('El area del Rectángulo es: %d', area)
      fprintf(fid,"\nRectangulo de base %d y altura %d tiene de area %d",b,h,area)
  else
      fprintf("saliendo calculadora")
  end
  fclose(fid)

File: areas.py
import math
import psycopg2

def cuadrado(L):
    conexion = psycopg2.connect(host="localhost", database="tarea", user="postgres")
    cursor1=conexion.cursor()
    sql="insert into areas(fig, area) values (%s,%s)"
    area=L**2
    f = open('areas.txt','a')
    f.write('\nlado' + str(L) + 'el area del cuadrado es: '+ str(area))
    f.close()
    datos=("cuadrado", area)
    cursor1.execute(sql, datos)
    conexion.commit()
    conexion.close() 
    return  print('el area del cuadrado es: ', area)

def circulo(R):
    conexion = psycopg2.connect(host="localhost", database="tarea", user="postgres")
    cursor1=conexion.cursor()
    sql="insert into areas(fig, area) values (%s,%s)"
    area=math.pi*R**2
    f = open('areas.txt','a')
    f.write('\nRadio' + str(R) + 'el area del circulo es: '+ str(area))
    f.close()
    datos=("circulo", area)
    cursor1.execute(sql, datos)
    conexion.commit()
    conexion.close() 
    return print('el area del circulo es ', area)

def triangulo(b,h):
    conexion = psycopg2.connect(host="localhost", database="tarea", user="postgres")
    cursor1=conexion.cursor()
    sql="insert into areas(fig, area) values (%s,%s)"
    area=b*h/2
    f = open('areas.txt','a')
    f.write('\nbase' + str(b) + 'altura' + str(h) + 'el area del triangulo es: '+ str(area))
    f.close()
    datos=("triangulo", area)
    cursor1.execute(sql, datos)
    conexion.commit()
    conexion.close() 
    return print('El area del triángulo es: ', area)

def rectangulo(b,h):
    conexion = psycopg2.connect(host="localhost", database="tarea", user="postgres")
    cursor1=conexion.cursor()
    sql="insert into areas(fig, area) values (%s,%s)"
    area=b*h
    f = open('areas.txt','a')
    f.write('\nbase' + str(b) + 'altura' + str(h) + 'el area del rectangulo es: '+ str(area))
    f.close()
    datos=("rectangulo", area)
    cursor1.execute(sql, datos)
    conexion.commit()
    conexion.close() 
    return print('El area del Rectángulo es: ', area)

def calculadora():
  print("""calculadora de areas para:
         1.Cuadrado.
         2.Circulo.
         3.Triángulo.
         4.Rectángulo.
         otro para Salir
      ****************************""")

  x=int(input("Escoja la figura: "))

  if x==1:
      L=int(input('Ingrese el lado: '))
      cuadrado(L)
    
  elif x==2:
      R=int(input('Ingrese el radio del circulo: '))
      circulo(R)
    
  elif x==3:  
      b=int(input('Ingrese la base: '))
      h=int(input('Ingrese la altura: '))
      triangulo(b,h)
    
  elif x==4:
      b=int(input('Ingrese la base: '))
      h=int(input('Ingrese la altura: '))
      rectangulo(b,h)
  else:
      print("no saliendo calculadora")
calculadora()
